pkg/sensors/manager.go

```diff
@@ -335,10 +325,16 @@ func (h *Manager) LogSensorsAndProbes(ctx context.Context) {
 	log.WithField("types", strings.Join(names, ", ")).Info("Registered probe types")
 }
 
+// policyLister allows read-only access to the collections map
+type policyLister interface {
+	listPolicies() []*tetragon.TracingPolicyStatus
+}
+
 // Manager handles dynamic sensor management, such as adding / removing sensors
 // at runtime.
 type Manager struct {
 	sensorCtl sensorCtlHandle
+	policyLister
```

Comment:
nit: as it might look a bit weird, maybe we can add a comment here to explain (but we have git history anyway).

Reply:
pushed a new version with a comment. thanks!